.changes/next-release/removal-AWSSDKforJavav2-c58118f.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "removal",
```

Comment:
Do we need a changelog entry given that this is just house keeping change?

Reply:
Will remove changelog